packages/frontend-shared/cypress/e2e/e2ePluginSetup.ts

```diff
@@ -155,42 +157,57 @@ async function makeE2ETasks () {
 
       sinon.stub(ctx.util, 'fetch').get(() => {
```

Comment:
IMO this all looks like a suitable place to use https://github.com/visionmedia/supertest and not have to write all this routing yourself, but this is fine too